### DISCUSSION TITLE
发送语音【抱歉】。

### DISCUSSION THREAD

**Discussion by fovj** [Pik-4/HsMod#201] [Ideas (许愿池)]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

否。

### Describe the solution you'd like/描述你想要的解决方案

![image](https://github.com/user-attachments/assets/569d19b1-d052-47d6-8f6a-26222a9954ad)

发送语音【抱歉】，单方面。

**Comment by Lofuuzi** [comment]
應該可以改成雙方
抱歉只是被隱藏了
